blueflood-http/src/integration-test/java/com/rackspacecloud/blueflood/inputs/handlers/HttpHandlerIntegrationTest.java

```diff
@@ -220,8 +336,48 @@ private URIBuilder getMetricsURIBuilder() throws URISyntaxException {
                 .setPort(httpPort).setPath("/v2.0/acTEST/ingest");
     }
 
+    private static void createAndInsertTestEvents(final String tenant, int eventCount) throws Exception {
+        ArrayList<Map<String, Object>> eventList = new ArrayList<Map<String, Object>>();
+        for (int i=0; i<eventCount; i++) {
+            Event event = new Event();
+            event.setWhat("deployment");
+            event.setWhen(Calendar.getInstance().getTimeInMillis());
+            event.setData("deploying prod");
+            event.setTags("deployment");
+
+            eventList.add(event.toMap());
+        }
+        eventsSearchIO.insert(tenant, eventList);
+    }
+
+    private HttpResponse postEvent(String requestBody, String tenantId) throws Exception {
```

Comment:
Is it possible to combine these helper methods with the helper methods in `HttpAnnotationsEndToEndTest`?